stage1/init/pod.go

```diff
@@ -474,6 +474,11 @@ func (p *Pod) appToNspawnArgs(ra *schema.RuntimeApp) ([]string, error) {
 
 	for _, v := range p.Manifest.Volumes {
 		vols[v.Name] = v
+		if v.Kind == "empty" {
+			if err := os.MkdirAll(filepath.Join(common.Stage1SharedVolumesDir(p.Root), v.Name.String()), 0755); err != nil {
```

Comment:
I am wondering about this `0755`. With `host` volumes, we don't need to specify the rights of the mount point because it comes from the existing directory on the host. But with `empty` volumes, maybe the manifest should tell us how the volume should be created (which owner and permissions).

Also, with #1452 coming soon, should we explicitly remove the `os.ModeSetgid` bit on the "sharedVolumes" directory?